### PR TITLE
typo: Adapater -> Adapter

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

**Review by mairaw — APPROVED** [review]
LGTM

**Review by mairaw — CHANGES_REQUESTED** [review]
Actually, it seems that this topic uses the variable adapter in the other parts of the code.

**Comment by nschonni** [comment]
The second example uses `adapter` throughout, but the first one uses 3 different undeclared variables :confused:

File: docs/framework/data/adonet/updating-data-sources-with-dataadapters.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ adapter.Update(table.Select(Nothing, Nothing, _
   DataViewRowState.ModifiedCurrent))  
   
 ' Finally, process inserts.  
-dataAdapater.Update(table.Select(Nothing, Nothing, _  
+dataAdapter.Update(table.Select(Nothing, Nothing, _
```
Comment (mairaw) [inline docs/framework/data/adonet/updating-data-sources-with-dataadapters.md]: I'd just change this to match the C# version

```suggestion
adapter.Update(table.Select(Nothing, Nothing, _  
```

**Comment by mairaw** [comment]
Please file an issue for the other issue you're reporting @nschonni so @stevestein can take a look.

File: docs/framework/data/adonet/updating-data-sources-with-dataadapters.md (hunk 1 above)
Comment (nschonni) [reply docs/framework/data/adonet/updating-data-sources-with-dataadapters.md]: Arg, the branch merge messed up the suggestion

File: docs/framework/data/adonet/updating-data-sources-with-dataadapters.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ adapter.Update(table.Select(Nothing, Nothing, _
   DataViewRowState.ModifiedCurrent))
 
 ' Finally, process inserts.
-dataAdapater.Update(table.Select(Nothing, Nothing, _
+dataAdapter.Update(table.Select(Nothing, Nothing, _
```
Comment (mairaw) [inline docs/framework/data/adonet/updating-data-sources-with-dataadapters.md]:
```suggestion
  adapter.Update(table.Select(Nothing, Nothing, _
```

File: docs/framework/data/adonet/updating-data-sources-with-dataadapters.md (hunk 1 above)
Comment (mairaw) [reply docs/framework/data/adonet/updating-data-sources-with-dataadapters.md]: I might have done something wrong when resolving the conflict. Did a new suggestion.

**Review by mairaw — APPROVED** [review]

**Pull request merged**